test/install.mk

```diff
@@ -119,10 +120,15 @@ install-egress:
 # Telemetry will be installed in istio-control for the tests, until integration tests are changed
 # to expect telemetry in separate namespace
 install-telemetry:
-	#bin/iop istio-telemetry istio-grafana $IBASE/istio-telemetry/grafana/ 
+	bin/iop ${ISTIO_NS} istio-grafana ${BASE}/istio-telemetry/grafana/  ${IOP_OPTS}
```

Comment:
Same for grafana - would be great to install it in istio-telemetry or even istio-grafana. 

Quite a few customers have their own grafana installs - I want to make sure grafana installed in another ns will work 
with istio, and I don't think we have tests checking grafana and assuming same namepace as istio.

Can be done in separate PR if you want.

Reply:
Will create separate PR for updating grafana related issue, including the istio gateway change